### PR TITLE
Change default of legacy_keylisting option in app.config to false.

### PR DESCRIPTION
**Pull request by kellymclaughlin**

### CODE REVIEW

File: rel/files/app.config [hunk 1]
```diff
@@ -132,11 +132,8 @@
             {vnode_vclocks, true},
 
             %% This option enables compatability of bucket and key listing
-            %% with 0.14 and earlier versions. Once a rolling upgrade to
-            %% a version > 0.14 is completed for a cluster, this should be 
-            %% set to false for improved performance for bucket and key
-            %% listing operations.
-            {legacy_keylisting, true}
+            %% with 0.14 and earlier versions.
```
Comment (Vagabond) [inline rel/files/app.config]: I think you removed a little too much from the comment here. The note saying that after the rolling upgrade is complete you should change this to false is useful information.

Comment (kellymclaughlin) [inline rel/files/app.config]: I thought maybe I was being too verbose. I'll add it back.

**Comment by Vagabond** [comment]
+1 on this.

**Comment by Vagabond** [comment]
This was merged but not closed on GH.

**Pull request closed without merging**